api/tests/test_user.py

```diff
@@ -347,5 +374,62 @@ def test_allocated_assets_are_returned(self, mock_verify_token):
             HTTP_AUTHORIZATION="Token {}".format(self.token_admin)
         )
         self.assertEqual(len(response.data['allocated_assets']), count + 1)
-        self.assertEqual(response.data['allocated_assets'][0].get('model_number'), 'IMN50987')
-        self.assertEqual(response.data['allocated_assets'][0].get('serial_number'), 'SN001')
+        self.assertEqual(
+            response.data['allocated_assets'][0].get('model_number'),
+            'IMN50987')
+        self.assertEqual(
+            response.data['allocated_assets'][0].get('serial_number'),
+            'SN001')
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_admin_user_filter_users_by_cohort(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.admin_user.email}
+        response = client.get(
+            '{}?cohort={}'.format(self.users_url, self.user.cohort),
+            HTTP_AUTHORIZATION="Token {}".format(self.token_admin))
+        self.assertEqual(response.data['results'][0]['cohort'],
+                         self.user.cohort)
+        self.assertEqual(response.data['count'], 2)
+        self.assertEqual(response.status_code, 200)
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_admin_user_filter_users_by_email(self, mock_verify_token):
+        mock_verify_token.return_value = {'email': self.admin_user.email}
+        response = client.get(
+            '{}?email={}'.format(self.users_url, self.user.email),
```

Comment:
Refactor tests related to filtering by email to emulate a letter in the alphabet as the passed value (case insensitive). 👇 

<img width="1638" alt="screen shot 2018-09-03 at 21 03 28" src="https://user-images.githubusercontent.com/10899666/44998278-df4edc80-afbc-11e8-9efa-fa1d090d8329.png">